src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] Please as it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md) for this project, " `Make sure to add test for all your components and functions.` ", kindly add tests for all your components and functions using React Testing Library and Jest as this will ensure your app doesn't break in production.